### ISSUE TITLE
Handlebars is rendering just first image

### ISSUE THREAD

**Issue by pofider** [jsreport/jsreport#145]
Handlebars engine is wrongly extracting image name in the following template. It just works for the first one
```html
<img src='{#image {{name}}}'/>
<img src='{#image {{name}}}'/>
```

Original report
http://jsreport.net/learn/images#comment-2348958203

Current workaround is to jsrender or child template with single image.

**Comment by bjrmatos** [comment]
fixed in https://github.com/jsreport/toner-handlebars/commit/fd62174f3da6b036dec088a5dbf624d805cf6e14

**Comment by pofider** [comment]
Released in 0.10
